.github/workflows/markdown-linter.yaml

```diff
@@ -0,0 +1,15 @@
+name: Markdown lint
+
+on:
+  workflow_call:
```

Comment:
its on workflow_dispatch you probably want to use. workflow_call allows another repository to call this workflow. workflow_dispatch allows you to trigger the pipeline manually.